magento/pwa-studio
# [PWA-809] Allow PWA Studio to use a different store to localize Catalog Products

## Description

Product data should be retrieved from the defined store view and displayed correctly on the product page and category pages.
Product prices should be in default store display currency, and format should be localized according to store locale, 

## Related Issue





https://jira.corp.magento.com/browse/PWA-809
https://jira.corp.magento.com/browse/PWA-810

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Create a Category and Product in the Magento admin
2. Configure Category and Product to have different data (description, meta title, etc ...) associated to a specific store view which has different locale setting
2. Add one more allowed currency for this store view
3. Change default display currency for this store. 
4. Modify the STORE_VIEW_CODE to point to that store
5. Verify the Category listing page is loaded from the correct store with all associated data localized.
6. Verify the Product details page is loaded from the correct store with all associated data localized.
7. Modify STORE_VIEW_CODE to both store views and verify store changes after a restart of the watch command
## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
